client/src/pages/CreateUserProfile.js

```diff
@@ -195,8 +155,21 @@ const CreateProfile = ({ email, history }) => {
     initialState,
   );
 
+  const handleInputChangePrivacy = (e) => {
+    setPrivacy(e.target.checked);
+  };
+  const handleInputChangeConditions = (e) => {
+    setConditions(e.target.checked);
+  };
+
   const onSubmit = async (formData) => {
-    if (!location.address) {
+    if (!privacy) {
+      alert("You must agree to our privacy policy before proceeding");
+      return;
+    } else if (!conditions) {
+      alert("You must agree to our terms and conditions before proceeding");
+      return;
+    } else if (!location) {
       // all location objects should have address (+coordinates), others optional
       return setError(
         "location",
```

Comment:
Minor: Maybe we can use multiple condition in ternary operator here instead of if statements or a switch case statement here?

Reply:
No necessarily, cuz you are not checking the if the value is one or the other, but unique values and their values..

Reply:
Alright sounds good then!